Mathlib/Data/PFunctor/Multivariate/Basic.lean

```diff
@@ -38,122 +38,125 @@ open MvFunctor (LiftP LiftR)
 variable {n m : ℕ} (P : MvPFunctor.{u} n)
 
 /-- Applying `P` to an object of `Type` -/
+@[coe]
```

Comment:
What does the `coe` attribute do? Since we also have the explicit `CoeFun` instance a few lines lower

Reply:
The function with a `coe` attribute is printed as `↑`.